### PR TITLE
Add `--upgrade` command to Schema Loader

### PR DESCRIPTION
**Pull request by Torch3333**
## Description

This adds a new `--upgrade` command to the Schema Loader, which directly calls the Admin API `upgrade()` method.
For more details about what `upgrade` does, please refer to https://github.com/scalar-labs/scalardb/pull/1204

## Related issues and/or PRs

- https://github.com/scalar-labs/scalardb/pull/1204

## Changes made

Adds a new `--upgrade` command to the Schema Loader

## Checklist

> The following is a best-effort checklist. If any items in this checklist are not applicable to this PR or are dependent on other, unmerged PRs, please still mark the checkboxes after you have read and understood each item.

- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

- The documentation will be updated in another PR

## Release notes

Added a new command `--upgrade` to the Schema Loader. Running this method when updating an existing environment to ScalarDB 4.X is necessary to support backward incompatible changes introduced by the new "namespaces management" feature. This command is strictly equivalent to the Admin API `upgrade()` method.

### CODE REVIEW

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

File: schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java [hunk 1]
```diff
@@ -575,6 +575,40 @@ private static void importTables(Either<Path, Properties> config, Either<Path, S
...
+   *
+   * @param configPath path to the ScalarDB config properties.
+   * @param options specific options for upgrading.
+   * @throws SchemaLoaderException thrown when upgrading failed.
+   */
+  public static void upgrade(Path configPath, Map<String, String> options)
+      throws SchemaLoaderException {
+    Either<Path, Properties> config = new Left<>(configPath);
+    upgrade(config, options);
+  }
```
Comment (feeblefakie) [inline schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java:591]: Super minor, but we probably need an empty line?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left one super minor comment.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java (hunk 1 above)
Comment (Torch3333) [reply schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java:591]: Indeed, thank you.
It's a shame Spotless does not enforce a blank line between method definitions.
https://github.com/scalar-labs/scalardb/pull/1208/commits/be5165f9652744402a98758aa98ecf7320b26d7f

**Pull request merged**